CONTRIBUTING.md

```diff
@@ -0,0 +1,255 @@
+
+    ```python
+    import numpy as np
+
+    from monai.config import print_config
+    from monai.data import DataLoader
+    ...
+
+    print_config()
+    ```
+
+The tutorial provides [useful templates](.github/contributing_templates/notebook/README.md) for contributors to start with.
+
+### Commit new changes
+
+MONAI enforces the [Developer Certificate of Origin](https://developercertificate.org/) (DCO) on all pull requests.
+All commit messages should contain the `Signed-off-by` line with an email address.
+The [GitHub DCO app](https://github.com/apps/dco) is deployed on MONAI.
+The pull request's status will be `failed` if commits do not contain a valid `Signed-off-by` line.
+
+Git has a `-s` (or `--signoff`) command-line option to append this automatically to your commit message:
+```bash
+git commit -s -m 'your awesome commit summary'
+```
+The commit message will be:
+```
+    your awesome commit summary
+
+    Signed-off-by: Your Name <yourname@example.org>
+```
+
+Full text of the DCO:
+```
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
+
+Everyone is permitted to copy and distribute verbatim copies of this
+license document, but changing it is not allowed.
+
+
+Developer's Certificate of Origin 1.1
+
+By making a contribution to this project, I certify that:
+
+(a) The contribution was created in whole or in part by me and I
+    have the right to submit it under the open source license
+    indicated in the file; or
+
+(b) The contribution is based upon previous work that, to the best
+    of my knowledge, is covered under an appropriate open source
+    license and I have the right under that license to submit that
+    work with modifications, whether created in whole or in part
+    by me, under the same open source license (unless I am
+    permitted to submit under a different license), as indicated
+    in the file; or
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+
+### Open a pull request
+
+Once the changes are ready, push your local branch to the remote, e.g. `git push --set-upstream origin <your new branch name>`, and [open a pull request](https://github.com/Project-MONAI/tutorials/pulls).
+When the pull request is opened, the MONAI repository has a set of GitHub actions that will run checks on the changes.
+
+Please check more details in the [guidelines](#ci-test-passing-guide) on how to pass the tests](#ci-test-passing-guide).
```

Comment:
Missed "[" in "on how to pass the tests]".

Thanks.

Reply:
Fixed. Thanks!